pagopa/io-app
# fix: [IOBP-786] Transaction total amount and total amount message

## Short description
This PR fixes the total amount when the transaction doesn't have a transaction fee and it adds an additional explanation label that advice the user that the total amount doesn't include the transaction fee.

## List of changes proposed in this pull request
- Added locales for the total fee unknown and total fee unknown without PSP;
- Added a check to verify if the backend provides us the amount fee, if it doesn't so it must show the explanation label;
- Fixed the calculation of the total amount, in case the transaction fee is undefined it must be treated as zero-value

## How to test
Start a payment with the io-dev-server and reach the end of the transaction. Try to edit the transaction fee from the dev-serve removing it from the response and check what happens
